test/runtest.py

```diff
@@ -2511,6 +2517,47 @@ def f():
     assert len(ready_ids) == 1
 
 
+def test_inline_objects(shutdown_only):
+    config = json.dumps({"initial_reconstruction_timeout_milliseconds": 200})
+    ray.init(num_cpus=1, object_store_memory=10**7, _internal_config=config)
+
+    @ray.remote
+    class Actor(object):
+        def create_inline_object(self):
+            return "inline"
+
+        def create_non_inline_object(self):
+            return 10000 * [1]
+
+        def get(self):
+            return
+
+    def flush(actor):
```

Comment:
The release delay is gone as of https://github.com/apache/arrow/pull/3124, so we can get rid of this, yay!

Reply:
Ah great, thanks!

Reply:
Hmm actually, if I remove the `flush` call, this test sometimes fails on my laptop. Do you know why that is (not sure about the semantics of `ray.internal.free` for plasma)?

Reply:
This is probably a race condition (ray.internal.free is using the plasma client in the raylet).

One way around that is to use ray.workers.global_worker.plasma_client.delete, which uses the same plasma client that will be used for the get, so that should work.

Reply:
Hmm this still seems to fail occasionally with `plasma_client.delete` (maybe 1/100 times). Any idea why or do you think it's a bug somewhere else?